core/modules/layout/tests/layout_test/layout_test.info

```diff
@@ -5,3 +5,8 @@ version = BACKDROP_VERSION
 type = module
 backdrop = 1.x
 hidden = TRUE
+
+; Added by Backdrop CMS packaging script on 2021-01-27
```

Comment:
Same thing here, no version info needed in core (will be added by the packager later)